src/gui/qgsprojectionselectiontreewidget.cpp

```diff
@@ -655,25 +656,28 @@ void QgsProjectionSelectionTreeWidget::loadCrsList( QSet<QString> *crsFilter )
       {
         // This is a projected srs
         QTreeWidgetItem *node = nullptr;
-        QString srsType = QString::fromUtf8( ( char * )sqlite3_column_text( stmt, 4 ) );
-        if ( srsType.isEmpty() )
-          srsType = tr( "Other" );
+        const QString projectionAcronym = QString::fromUtf8( ( char * )sqlite3_column_text( stmt, 4 ) );
+        QString projectionName;
+        if ( projectionAcronym.isEmpty() )
+          projectionName = tr( "Other" );
+        else
+          projectionName = QgsCoordinateReferenceSystemUtils::translateProjection( projectionAcronym );
 
         // Find the node for this type and add the projection to it
         // If the node doesn't exist, create it
-        if ( srsType == previousSrsType )
+        if ( projectionName == previousSrsType )
```

Comment:
I'd change the previousSrsType variable to previousProjectionName, easier to read for people who will visit the code in years to come :)

Reply:
Done (but actually this code is about to be moved to the ether anyway!)